### PR TITLE
Add simple regexp match for multiple prompt

### PR DESCRIPTION
**Pull request by Landrash**
##### SUMMARY
Add a simple regexp match for multiple prompt with multiple answers.
This example could be used to for restarting a network device with a delay.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
ansible.netcommon.cli_command

##### ADDITIONAL INFORMATION
Matches prompt with regexp instead of full prompt to simplify multiple prompts with dynamic och convoluted output.

### CODE REVIEW

File: plugins/modules/cli_command.py
Comment (Qalthos) [inline plugins/modules/cli_command.py:123]: You have an indentation error here
```suggestion
- name: Simple regexp match for multiple prompt, multiple answer(mandatory check for all prompts)
  ansible.netcommon.cli_command:
    command: reload in 5
    check_all: true
    prompt:
    - Save\?
    - confirm
    answer:
    - n
    - y
```

**Review by Qalthos — CHANGES_REQUESTED** [review]
Can you please add a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-changelog-fragments) to your PR? The `doc_changes` section should be appropriate.

Thanks!

**Comment by Landrash** [comment]
Added changelog fragment and synced branch with upstream to try and fix build failure.

File: changelogs/fragments/cli-command-module.yaml
Comment (Qalthos) [inline changelogs/fragments/cli-command-module.yaml:2]:
```suggestion
doc_changes:
```

Comment (Qalthos) [inline changelogs/fragments/cli-command-module.yaml:2]:
```suggestion
doc_changes:
```

**Review by Qalthos — CHANGES_REQUESTED** [review]

**Review by Qalthos — APPROVED** [review]

File: plugins/modules/cli_command.py
Comment (NilashishC) [reply plugins/modules/cli_command.py:1]: @Landrash Hi! Just a heads up - we are still waiting for you to make the requested changes before we can merge this PR.

**Pull request merged**
